### ISSUE TITLE
YAML errors should report before @INC errors

### ISSUE THREAD

**Issue by SleepyJay** [gryphonshafer/Config-App#5]
If using `libs` in a YAML and that YAML doesn't parse, you will get error: `Can't locate <thing> in @INC...`. This may not alert you to bad YAML. (This also might be really hard to fix?)

Test:
1. Have YAML like this (note the space before `blah` for error):
```
default:
  libs: ./lib
 blah:
```  
2. Have wrapper file like this:
```
use Config::App;
use Foo::Bar;
```

3. Note `@INC` error.

4. Success: if you remove the bad YAML part and actually have a compiling module at `lib/Foo/Bar.pm`, the above should work.

**Comment by gryphonshafer** [comment]
This issue will likely be resolved with #4 fix.

**Comment by gryphonshafer** [comment]
Resolved in 1e0f0ef.

**Comment by SleepyJay** [comment]
(Confirmed fixed, FYI)
